docs/_dapp/native-accounts.md

```diff
@@ -3,140 +3,107 @@ title: Go Account Management
 sort_key: D
 ---
 
-To provide Ethereum integration for your native applications, the very first thing you
-should be interested in doing is account management.
-
-Although all current leading Ethereum implementations provide account management built in,
-it is ill advised to keep accounts in any location that is shared between multiple
-applications and/or multiple people. The same way you do not entrust your ISP (who is
-after all your gateway into the internet) with your login credentials; you should not
-entrust an Ethereum node (who is your gateway into the Ethereum network) with your
-credentials either.
-
-The proper way to handle user accounts in your native applications is to do client side
-account management, everything self-contained within your own application. This way you
-can ensure as fine grained (or as coarse) access permissions to the sensitive data as
-deemed necessary, without relying on any third party application's functionality and/or
-vulnerabilities.
-
-To support this, `go-ethereum` provides a simple, yet thorough accounts package that gives
-you all the tools to do properly secured account management via encrypted keystores and
-passphrase protected accounts. You can leverage all the security of the `go-ethereum`
-crypto implementation while at the same time running everything in your own application.
+Geth provides a simple, yet thorough accounts package that includes all the tools developers
+need to leverage all the security of Geth's crypto implementation in a Go native application.
+The account management is done client side with all sensitive data held inside the application.
+This gives the user control over access permissions without relying on any third party.
```

Comment:
Can you add something here about how `Clef` is the recommended way to manage keys? We plan to phase out native accounts over the long run

Reply:
Sure - done!